docs/vision-usage.rst

```diff
@@ -0,0 +1,233 @@
+Using the Vision API
+====================
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If the GCLOUD_PROJECT environment variable is not present,
+  the project ID from JSON file credentials is used.
+
+  If you are using Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.vision.client.Client>`
+
+.. code-block:: python
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+or pass in ``credentials`` and ``project`` explicitly
+
+.. code-block:: python
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.png')
+    >>> faces = image.detect_faces(limit=10)
+
+Annotate multiple images
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+     >>> first_image = client.image('./image.jpg')
+     >>> second_image = client.image('gs://my-storage-bucket/image2.jpg')
+     >>> with client.batch():
+     ...     labels = first_image.detect_labels()
+     ...     faces = second_image.detect_faces(limit=10)
+
+or
+
+.. code-block:: python
+
+     >>> images = []
+     >>> images.append(client.image('./image.jpg'))
+     >>> images.append(client.image('gs://my-storage-bucket/image2.jpg'))
+     >>> faces = client.detect_faces_multi(images, limit=10)
```

Comment:
Definitely could, and I think this will change a bit in general and get more consistent. I played with the GAX stuff last week and I want to try get the JSON implementation working very close to how GAX does batching.
Your thought here is very close to how the GAX stuff works.